deployment/terraform/environments/oss-vdb-test/main.tf

```diff
@@ -3,7 +3,8 @@ module "osv_test" {
 
   project_id = "oss-vdb-test"
 
-  public_import_logs_bucket = "osv-test-public-import-logs"
+  public_import_logs_bucket  = "osv-test-public-import-logs"
+  osv_vulnerabilities_bucket = "osv-test-vulnerabilities"
```

Comment:
nit: let's rename this to "vulnerabilities_export_bucket" to be clearer what this is.